src/OpenIddict.Abstractions/Primitives/OpenIddictExtensions.cs

```diff
@@ -3097,6 +3097,12 @@ private static bool IsEmptyJsonElement(JsonElement element)
     {
         switch (element.ValueKind)
         {
+            case JsonValueKind.Undefined:
+                return true;
+
+            case JsonValueKind.Null:
+                return true;
```

Comment:
For bonus points, we can merge the two cases 😄 

```suggestion
            case JsonValueKind.Null or JsonValueKind.Undefined:
                return true;
```